db/migrate/20170315095936_update_persistent_volumes_parent_type.rb

```diff
@@ -0,0 +1,7 @@
+class UpdatePersistentVolumesParentType < ActiveRecord::Migration[5.0]
+  def change
+    PersistentVolume.all.each do |pv|
```

Comment:
thanks @cben 

STI-disabling: commented here https://github.com/ManageIQ/manageiq/pull/14320#pullrequestreview-27307769 

@cben it is not ok but I did not figured out yet why it is not failing on CI( https://github.com/ManageIQ/manageiq/pull/10050 )